Kitodo/src/main/java/org/kitodo/config/Parameters.java

```diff
@@ -0,0 +1,688 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.config;
+
+/**
+ * These constants define configuration parameters usable in the configuration
+ * file. This file reflects the order of the global
+ * {@code kitodo_config.properties} file (as far as possible), and vice versa.
+ */
+public class Parameters {
+
+    /*
+     * =========================================================================
+     * ==== FILE AND DIRECTORY MANAGEMENT
+     * =========================================================================
+     * ==== ----------------------------------- Directories
+     * -----------------------------------
+     */
+
+    // DIR_XML_CONFIG = "KonfigurationVerzeichnis" → org.kitodo.config.Config
+
+    /**
+     * Absolute path to the directory that the rule set definition files will be
+     * read from. It must be terminated by a directory separator ("/").
+     */
+    public static final String DIR_RULESETS = "RegelsaetzeVerzeichnis";
+
+    /**
+     * Absolute path to the directory that XSLT files are stored in which are
+     * used to transform the "XML log" (as visible from the XML button in the
+     * processes list) to a downloadable PDF docket which can be enclosed with
+     * the physical binding units to digitise. The path must be terminated by a
+     * directory separator ("/").
+     */
+    public static final String DIR_XSLT = "xsltFolder";
+
+    // DIR_PROCESSES = "MetadatenVerzeichnis" → org.kitodo.config.Config
+
+    /**
+     * Absolute path to the base directory of the users' home directories,
+     * terminated by a directory separator ("/"). If a user accepts a task to
+     * work on which will require him or her to have access permission to the
+     * data of a process, a symbolic link to the process directory in question
+     * will be created in his or her home directory that will be removed again
+     * after finishing the task. Note: If LDAP is used, the users' home dirs
+     * will instead be read from LDAP
+     */
+    public static final String DIR_USERS = "dir_Users";
+
+    /**
+     * Absolute path to a folder the application can temporarily create files
+     * in, terminated by a directory separator ("/").
+     */
+    public static final String DIR_TEMP = "tempfolder";
+
+    /**
+     * Path to directory in which BPMN diagrams are stored.
+     */
+    public static final String DIR_DIAGRAMS = "diagramsFolder";
+
+    // DIR_MODULES = "moduleFolder" → org.kitodo.config.Config
```

Comment:
Why did you add this out commented line? If this line is not needed, then do not add them or add at least an explanation to it.